src/Client.php

```diff
@@ -214,55 +232,64 @@ public function response(ResponseInterface $response)
 	}
 
 	/**
-	 * @return array
+	 * @return string[]
```

Comment:
@arnar-peturs this type hint is wrong it should be `array<string, string>`

`string[]` means `array<int, string>`